app/views/style_guides/reports.haml

```diff
@@ -0,0 +1,95 @@
+
+:scss
+  .c-indicator {
+    min-height: 75px;
+    margin: 0 auto;
+  }
+  .c-indicator--constrained-width {
+    min-width: 300px;
+    max-width: 500px;
+    margin: 0 auto;
+  }
+  .c-indicator__title {
+    font-weight: bold;
+    margin-bottom: 10px;
+  }
+  .c-indicator__content {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+  .c-indicator__passed {
+    margin-right: 50px;
+    span {
+      font-size: 35px;
+    }
+    &.pass {
+      color: green;
+    }
+    &.fail {
+      color: red;
+    }
+  }
+  .c-indicator__values {
+    display: flex;
+    align-items: flex-end;
+  }
+  .c-indicator__value {
+    margin-right: 10px;
+  }
+  .c-indicator__change-icon {
+    margin-left: 10px;
+    margin-right: 10px;
+    svg,
+    .icon-svg {
+      width: 2.5rem;
+      height: 2.5rem;
+    }
+  }
+  .c-indicator__value-number {
+    font-size: 40px;
+    font-weight: bold;
+    line-height: 1;
+  }
+  .c-indicator__value-unit {
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 1;
+  }
+  .c-indicator__value--secondary {
+    .c-indicator__value-number {
+      font-size: 20px;
+      font-weight: bold;
+    }
+    .c-indicator__value-unit {
+      font-size: 16px;
+    }
+  }
+  .c-indicator__label {
+    font-size: 15px;
+    line-height: 16px;
+  }
```

Comment:
@jgoley shouldn't all of this be in the main CSS?  (I'm confused why it's inline here)

Reply:
My bad — I inlined it for quicker iterations. I'll drop it in the styles as a component partial.